src/main/java/com/impossibl/postgres/jdbc/PGResultSet.java

```diff
@@ -2055,9 +2061,15 @@ int move(String type, Object loc) throws SQLException {
 
     result = null;
 
-    String sql = String.format("MOVE %s %s IN %s", type, loc, cursorName);
+    StringBuilder sb = new StringBuilder();
```

Comment:
I'll take those in a future PR - first concern was to get rid of String.format() which is dead slow, and hence shows up in performance profiles